src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -111,7 +111,7 @@ class CellFunctor {
    * Min. number of particles to start sorting.
    * @todo Currently, this is disabled because of https://github.com/AutoPas/AutoPas/issues/418
    */
-  constexpr static unsigned long _startSorting = std::numeric_limits<unsigned long>::max();
+  constexpr static unsigned long _startSorting = 1;
```

Comment:
Previously this threshold was set to 8 since sorting was not worth it for fewer particles. Maybe run some tests and see if this is still a reasonable heuristic.

Reply:
See PR description.